staging/src/k8s.io/apiserver/pkg/admission/metrics/filter.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission"
+)
+
+type LabelCardinalityFilter []webhookadmission.Rule
+
+func (f LabelCardinalityFilter) matches(gvr schema.GroupVersionResource, ns string) bool {
+	for _, rule := range f {
+		// namespace
+		namespace := len(rule.Namespaces) == 0
+		for _, n := range rule.Namespaces {
+			if n == "*" || n == ns {
```

Comment:
permitting "*" makes me a bit nervous..

Reply:
The does not expand cardinality, (at least not on its own). I just lets someone target resources in a specific namespace. For example: partition `pods` in the `system` namespace in the slo metric for monitoring specific to them.

Reply:
I have removed the namespace filter completely.